contrib/packaging/bootupd.spec

```diff
@@ -2,14 +2,14 @@
 
 %global crate bootupd
 
-Name:           bootupd
+Name:           rust-%{crate}
```

Comment:
this will not create a big issue in Fedora/RHEL? Changing the package name?

Reply:
This is just the *source* name, and it's already named this in Fedora https://src.fedoraproject.org/rpms/rust-bootupd but it isn't in RHEL https://gitlab.com/redhat/centos-stream/rpms/bootupd

Reply:
See https://github.com/coreos/bootupd/pull/838 which introduced this change

Reply:
IOW, one pain point we are going to need to eat here is that because it'd be hugely painful (AFAIK) to rename the *source* package in RHEL, we will have spec file divergence for the forseeable future there.